src/www/diag_confbak.php

```diff
@@ -57,6 +54,8 @@
         }
     }
 
+    $cnf = OPNsense\Core\Config::getInstance();
+    $confvers = $cnf->getBackups(true);
```

Comment:
does this work, because the loop accesses the variable defined here…

Reply:
would be nice if you could help me get rid auf the failed pr´s i did -> see https://github.com/opnsense/core/pull/2711